ssl/statem/extensions.c

```diff
@@ -890,40 +1124,77 @@ int tls_construct_extensions(SSL_CONNECTION *s, WPACKET *pkt,
         return 0;
     }
```

Comment:
So we're not doing anything special with the custom extensions?

Reply:
Yep, current code is a per the plan: https://github.com/openssl/openssl/blob/feature/ech/doc/designs/ech-api.md#handling-custom-extensions

Meaning: on the basis that we/I know of no "sensitive" custom ext types, and that the APIs required to do otherwise would be v. complex, custom ext values are visible in outer CH, ECH-compressed into inner CH. Doing something more complicated could be the subject of a future PR if needed.